force-app/main/default/classes/RD2_ChangeHistoryService_TEST.cls

```diff
@@ -34,7 +34,7 @@
 * @description All logic specific to Enhanced Recurring Donation Change History
 *
 */
-@IsTest(IsParallel=true)
+@IsTest(IsParallel=false)
```

Comment:
You don't need to specify `isParallel=false` since that's the default behavior.

Reply:
You know, I thought that, but I saw this in a different Test class, so **then** I thought something had changed 🤷‍♂️